src/ipelib/empirical_efield/efield_ipe.f

```diff
@@ -250,66 +262,42 @@ subroutine get_efield(i_rank)
 ! Author: A. Maute Dec 2003  am 12/17/03    
 !-----------------------------------------------------------------------
 
-!     use time_manager,   only : get_curr_calday, get_curr_date
-!     use mo_solar_parms, only : get_solar_parms
-!     use mag_parms,      only : get_mag_parms
-!     use cam_control_mod, only: magfield_fix_year
-!     use spmd_utils,      only: masterproc
-
-      integer :: i_rank
+      integer, intent(in):: i_rank
       integer :: idum1, idum2, tod ! time of day [s] 
       real kp
 
+
 !-----------------------------------------------------------------------
 ! get current calendar day of year & date components 
 ! valid at end of current timestep
 !-----------------------------------------------------------------------
-!     iday = get_curr_calday()                   ! day of year
-!     call get_curr_date (iyear,imo,iday_m,tod)! year, time of day [sec]
-!     iyear = magfield_fix_year
-!     iyear = 1995
-
-!     if( iyear < 1900 ) then
-!       write(iulog,"(/,'>>> get_efield: year < 1900 not possible: 
-!    &year=',i5)") iyear
-!       call endrun
-!     end if
 
       tod=ut*3600.
 !     ut = tod/3600.                   ! UT of day [sec]
-
-!-----------------------------------------------------------------------
-! get solar parms
-!-----------------------------------------------------------------------
-!     call get_solar_parms( f107_s = f107d )
-!-----------------------------------------------------------------------
-! get mag parms
-!-----------------------------------------------------------------------
-!     call get_mag_parms( by = by, bz = bz )
-!     print*,by,bz,f107d,ut
-!#ifdef EFIELD_DIAGS
-!      if( masterproc ) then
-!         write(iulog,*) 'get_efield: f107d,by,bz = ', f107d,by,bz 
-!      end if
-!#endif
 !-----------------------------------------------------------------------
 ! ajust S_a
 !-----------------------------------------------------------------------
-      call adj_S_a(i_rank)
+      call adj_S_a
 !-----------------------------------------------------------------------
 ! calculate global electric potential    
 !-----------------------------------------------------------------------
-      call GlobalElPotential(i_rank)
+      call GlobalElPotential
+!     print*,'pot_efield',potent(149,66),potent(149,64)
 
 !-----------------------------------------------------------------------
 ! calculate derivative of global electric potential 
 !-----------------------------------------------------------------------
       call DerivPotential
-!     print*,'ed2_efield',ed2(149,65),potent(149,66),potent(149,64)
 
-      end subroutine get_efield
+!      if(i_rank.eq.0) then
+!           write(2000+i_rank, *) potent
+!           write(2000+i_rank, *) ed1 
+!           write(2000+i_rank, *) ed2
+!      end if 
```

Comment:
overall the code changes read fine, would just clean up some of the new commented debug blocks in efield_ipe.f -- if there are print or write statements that are not needed generally, they should not be included in the pull request.

Reply:
already deleted.